recipe/meta.yaml

```diff
@@ -16,17 +16,23 @@ build:
     - vc9     # [win and py27]
     - vc10    # [win and py34]
     - vc14    # [win and py35]
+    - vc14    # [win and py36]
+  skip: true  # [win and py35]
 
 requirements:
   build:
+    - krb5 1.14.*
+    - libssh2 1.8.*
     # python is a build requirement on Windows to resolve features.
     - python           # [win]
     # perl is required to run the tests on UNIX.
-    - perl             # [unix]
+    - perl 5.22.2.1    # [unix]
     - pkg-config       # [unix]
     - openssl 1.0.*    # [unix]
     - zlib 1.2.*
   run:
+    - krb5 1.14.*
+    - libssh2 1.8.*
```

Comment:
I am not sure as to what is the exact policy on verifying pinnings (perhaps there is a doc on conda-forge which I missed)? According to https://abi-laboratory.pro/tracker/timeline/krb5/ , it seems like the pinning for krb5 seems to be correct, and according to https://abi-laboratory.pro/tracker/timeline/libssh2/ the pinning for libssh2 seems to harsh (perhaps we should pin on major?) I am happy to send a PR to update the pinning script, once we finalize on them.